frontend/src/hooks/api/getters/useFeature/useFeature.ts

```diff
@@ -25,16 +26,25 @@ export const useFeature = (
 ): IUseFeatureOutput => {
     const path = formatFeatureApiPath(projectId, featureId);
 
+    const { uiConfig } = useUiConfig();
+    const {
+        flags: { variantsPerEnvironment },
+    } = uiConfig;
+
     const { data, error, mutate } = useSWR<IFeatureResponse>(
         ['useFeature', path],
-        () => featureFetcher(path),
+        () => featureFetcher(path, variantsPerEnvironment),
         options
     );
 
     const refetchFeature = useCallback(() => {
         mutate().catch(console.warn);
     }, [mutate]);
 
+    useEffect(() => {
+        mutate();
+    }, [mutate, variantsPerEnvironment]);
+
```

Comment:
This should hopefully prevent a rare issue where the flag was not enabled on time for the fetch to happen, so we would fetch the feature without the query param below (no variants on screen).